OData2Poco.CommandLine.Test/OData2Poco.CommandLine.Test.csproj

```diff
@@ -42,8 +44,12 @@
       <HintPath>..\packages\Newtonsoft.Json.8.0.3\lib\net45\Newtonsoft.Json.dll</HintPath>
       <Private>True</Private>
     </Reference>
-    <Reference Include="nunit.framework, Version=2.6.4.14350, Culture=neutral, PublicKeyToken=96d09a1eb7f44a77, processorArchitecture=MSIL">
-      <HintPath>..\packages\NUnit.2.6.4\lib\nunit.framework.dll</HintPath>
```

Comment:
This reference was not correct because it misses the platform folder name. Therefore I couldn't compile the solution. So, I executed:
`update-package -Id nunit -reinstall`
in the package manager console to fix this.
This works fine, but apparently breaks the automated builds :-(